src/adapter/src/flags.rs

```diff
@@ -216,6 +218,21 @@ fn persist_config(config: &SystemVars) -> PersistParameters {
     }
 }
 
+pub fn pg_timstamp_oracle_config(config: &SystemVars) -> PostgresTimestampOracleParameters {
+    PostgresTimestampOracleParameters {
+        pg_connection_pool_max_size: Some(config.pg_timestamp_oracle_connection_pool_max_size()),
+        pg_connection_pool_max_wait: Some(config.pg_timestamp_oracle_connection_pool_max_wait()),
+        pg_connection_pool_ttl: Some(config.pg_timestamp_oracle_connection_pool_ttl()),
+        pg_connection_pool_ttl_stagger: Some(
+            config.pg_timestamp_oracle_connection_pool_ttl_stagger(),
+        ),
+        // WIP: Should we use our own here? Persist also uses these generically
```

Comment:
Good question! Personally, I'd lean toward consistency in the sense of "share all of them or share none of them"

Reply:
Meaning the pg oracle shouldn't share any with persist? And the current state where persist shares these `crdb_foo` vars with the stash is an anomaly?

Reply:
Hmm dunno! Mind digging up the PRs where this all was added to see if it was on purpose or not?

Reply:
This is the PR where Paul added the first `crdb_foo` system var: https://github.com/MaterializeInc/materialize/pull/17852#discussion_r1119212948, with a request from Matt to make the name generic/re-usable.

And this is where the usage in Stash was added: https://github.com/MaterializeInc/materialize/issues/17859.

afaict, `crdb_connect_timeout` is the only setting that's shared, and has a generic name. `crdb_tcp_user_timeout` _has_ a generic name but is not shared. The other persist settings have a `persist_` prefix and are not shared.

My gut feeling is that the ts oracle and persist have different usage patterns and so none of the vars should be shared. But the same might have been said about persist and Stash, so ... :shrug:

Reply:
Paul confirmed offline that the right thing to do was share crdb_connect_timeout and crdb_tcp_user_timeout and not share the others. So sounds like we can resolve this WIP?